### ISSUE TITLE
Adding response interceptor breaks proxy

### ISSUE THREAD

**Issue by elirud** [wkeeling/selenium-wire#375]
Trying to simply print my response status codes I added a response interceptor.

`def res_interceptor(request, response):  
    print(response.status_code)`

 After adding it the the page wont load anymore. The response interceptor does print the status code (200), but the page stops loading. Changing my interceptor to just pass causes the same issue. Refreshing the site in the browser after it initially stops loading gives ERR_PROXY_CONNECTION_FAILED.

**Comment by wkeeling** [comment]
Many thanks for raising this. I can also reproduce so I'll figure out what's happening.

**Comment by wkeeling** [comment]
Yes there's a bug here. I'll fix this ASAP. In the meantime you may be able to work around it by setting the `disable_encoding` option in the seleniumwire_options:

```
driver = webdriver.Chrome(seleniumwire_options={'disable_encoding': True})
```

**Comment by wkeeling** [comment]
Fixed in 4.5.2
